src/include/sof/math/sqrt.h

```diff
@@ -0,0 +1,17 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2021 Intel Corporation. All rights reserved.
+ *
+ * Author: Shriram Shastry <malladi.sastry@linux.intel.com>
+ *
+ */
+
+#ifndef __SOF_MATH__SQRTLOOKUP__H
+#define __SOF_MATH__SQRTLOOKUP__H
+
+/* Include Files */
+#include "stdint.h"
+
+/* Function Declarations */
+int32_t sqrt_int16(uint16_t u);
```

Comment:
Should function return type be int16_t or even uint16_t ?

Reply:
`sqrt_int16()` is  designed for a positive return value . There is no intention of negative value for a case `int16_t` ( sqrt algo is neither dealing with complex number / principal square root) . so its safe to `uint16_t`